server/src/model/activist.go

```diff
@@ -1748,6 +1765,46 @@ func GetSupporterSpokeInfo(db *sqlx.DB, chapterID int, startDate, endDate string
 	return activists, nil
 }
 
+func GetNewActivistSpokeInfo(db *sqlx.DB, chapterID int, startDate, endDate string) ([]ActivistSpokeInfo, error) {
+	query := `
+		SELECT
+			IF(preferred_name <> '', preferred_name, substring_index(name, " ", 1)) as first_name,
+			SUBSTRING(name, LOCATE(' ', name)+1) as last_name,
+			phone as cell,
+		    @last_event := (
+					SELECT MAX(e.date) as max_date
+					FROM event_attendance ea
+					JOIN events e ON ea.event_id = e.id
+					WHERE ea.activist_id = activists.id
```

Comment:
my old comment got outdated but did you want to extract a variable for this? since it's a whole query on its own and duplicated just a few lines below.